x-pack/plugins/cloud_security_posture/common/types.ts

```diff
@@ -85,6 +85,7 @@ export interface BaseCspSetupStatus {
   latestPackageVersion: string;
   cspm: BaseCspSetupBothPolicy;
   kspm: BaseCspSetupBothPolicy;
+  vuln_mgmt?: BaseCspSetupBothPolicy; // Todo: remove this when vuln_mgmt is ready
```

Comment:
I meant when vuln_mgmt onboarding prompt feature is ready. I was afraid that I would break any existing interfaces CSPStatusCode, but todo comment is unnecessary and a safety navigator. I was able to find all cases for the `BaseCspSetupBothPolicy` .